### PR TITLE
Update dkms.md

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

File: docs/guides/dkms.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ If you have a Touchbar, is it working? If no, then you'll need the apple-ibridge
 
 In some distros like Ubuntu (if using ISO with a kernel version `5.8.x` or below in case of Ubuntu as newer ISOs have got the updated drivers) you might need to uninstall the pre-installed BCE module and apple-ibridge module and install the ones given here as the pre-installed ones are not compatible with newer kernels (5.10+) but the ones given here are.
 
+In case you are installing a new kernel to get wifi working as described in the [wifi guide](https://wiki.t2linux.org/guides/wifi/) and also need to uninstall old drivers to get new ones, then you will have to follow this guide before installing the new kernel as new kernels need new drivers. In such a case you will have to use an ethernet cable, wifi adapter or phone tethering as this guide requires internet.
+
```
Comment (networkException) [inline docs/guides/dkms.md:13]: Couldn't this just be summed up to "remove any old modules that do not match the ones you are going to install now"?

Comment (AdityaGarg8) [reply docs/guides/dkms.md:13]: Yeah. You may edit it.

Comment (AdityaGarg8) [reply docs/guides/dkms.md:13]: Though I feel it more clear that way

Comment (networkException) [reply docs/guides/dkms.md:13]: I don't think its clear to have that many exceptions when to apply some changes. It doesn't matter why some modules are outdated, if they are outdated you should update them.

**Pull request closed without merging**